search_events_app/tests/models/test_feature.py

```diff
@@ -6,10 +6,7 @@
 class TestFeature(TestCase):
 
     def setUp(self):
-        self.feature = Feature.objects.create(name='Embedded Checkout', code='EC')
+        self.feature = Feature.objects.create(name='Website Widgets', code='WW')
```

Comment:
I think there is no necessary to put this line in "setUp" because you only have one test...